contracts/test/v0.8/foundry/vrf/VRFV2Plus.t.sol

```diff
@@ -0,0 +1,364 @@
82018054845181870281018701909552808552919492938584019390929083018282801561087d57602002820191906000526020600020905b815481526020019060010190808311610869575b5050509183525050600282015460ff81161515602083015261010090046001600160a01b0316604082015260039091015460609091015280519091506109055760405162461bcd60e51b815260206004820152601760248201527f7265717565737420494420697320696e636f7272656374000000000000000000604482015260640161027d565b6000838152600560209081526040909120835161092a92600190920191850190610b24565b5060008381526005602052604090819020600201805460ff191660011790556004805491517ffd26ba4b0000000000000000000000000000000000000000000000000000000081526001600160a01b039092169163fd26ba4b916109949187910190815260200190565b60206040518083038186803b1580156109ac57600080fd5b505afa1580156109c0573d6000803e3d6000fd5b505050506040513d601f19601f820116820180604052508101906109e49190610d5e565b6bffffffffffffffffffffffff166005600085815260200190815260200160002060030181905550505050565b6000546001600160a01b03163314610a6b5760405162461bcd60e51b815260206004820152601660248201527f4f6e6c792063616c6c61626c65206279206f776e657200000000000000000000604482015260640161027d565b565b6001600160a01b038116331415610ac65760405162461bcd60e51b815260206004820152601760248201527f43616e6e6f74207472616e7366657220746f2073656c66000000000000000000604482015260640161027d565b6001805473ffffffffffffffffffffffffffffffffffffffff19166001600160a01b0383811691821790925560008054604051929316917fed8889f560326eb138920d842192f0eb3dd22b4f139c87a2c57538e05bae12789190a350565b828054828255906000526020600020908101928215610b5f579160200282015b82811115610b5f578251825591602001919060010190610b44565b50610b6b929150610b6f565b5090565b5b80821115610b6b5760008155600101610b70565b803563ffffffff81168114610b9857600080fd5b919050565b600060208284031215610baf57600080fd5b81356001600160a01b0381168114610bc657600080fd5b9392505050565b600060208284031215610bdf57600080fd5b5035919050565b600060208284031215610bf857600080fd5b5051919050565b60008060408385031215610c1257600080fd5b8235915060208084013567ffffffffffffffff80821115610c3257600080fd5b818601915086601f830112610c4657600080fd5b813581811115610c5857610c58610dd0565b8060051b604051601f19603f83011681018181108582111715610c7d57610c7d610dd0565b604052828152858101935084860182860187018b1015610c9c57600080fd5b600095505b83861015610cbf578035855260019590950194938601938601610ca1565b508096505050505050509250929050565b60008060008060008060c08789031215610ce957600080fd5b863567ffffffffffffffff81168114610d0157600080fd5b9550610d0f60208801610b84565b9450604087013561ffff81168114610d2657600080fd5b9350610d3460608801610b84565b92506080870135915060a08701358015158114610d5057600080fd5b809150509295509295509295565b600060208284031215610d7057600080fd5b81516bffffffffffffffffffffffff81168114610bc657600080fd5b6020808252825182820181905260009190848201906040850190845b81811015610dc457835183529284019291840191600101610da8565b50909695505050505050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fdfea164736f6c6343000806000a";
```

Comment:
Why do this instead of just `new VRFV2PlusConsumerExample(...)`? It seems like this would change depending on optimizer settings and the code itself.